specification/purview/data-plane/Azure.Analytics.Purview.Account/readme.md

```diff
@@ -38,3 +46,104 @@ These settings apply only when `--tag=package-2019-11-01-preview` is specified o
 input-file:
 - preview/2019-11-01-preview/account.json
 ```
+
+---
+# Code Generation
+
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-go
+```
+
+
+## C# 
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+``` yaml $(csharp)
+csharp:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  namespace: Microsoft.Azure.Management.ProjectBabylon
+  output-folder: $(csharp-sdks-folder)/ProjectBabylon/Management.ProjectBabylon/Generated
+  clear-output-folder: true
+```
+
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
+
+``` yaml $(python)
+python-mode: create
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
+  namespace: azure.mgmt.ProjectBabylon
+  package-name: azure-mgmt-ProjectBabylon
+  clear-output-folder: true
+```
+``` yaml $(python) && $(python-mode) == 'update'
+python:
+  no-namespace-folders: true
+  output-folder: $(python-sdks-folder)/azure-mgmt-ProjectBabylon/azure/mgmt/ProjectBabylon
+```
+``` yaml $(python) && $(python-mode) == 'create'
+python:
+  basic-setup-py: true
+  output-folder: $(python-sdks-folder)/azure-mgmt-ProjectBabylon
+```
+
+
+## Go
```

Comment:
When you add a section for Go, you'll also need to add the appropriate `readme.go.md` file. If you don't need to release for Go right now (which seems likely), I believe removing this section will cause CI to skip building Go.

Reply:
added readme.go.md